tests/legacy-cli/e2e/tests/build/build-app-shell.ts

```diff
@@ -7,12 +7,12 @@ import { readNgVersion } from '../../utils/version';
 
 export default function() {
   // Skip this test in Angular 2/4.
-  if (getGlobalVariable('argv').ng2 || getGlobalVariable('argv').ng4) {
+  const argv = getGlobalVariable('argv');
+  if (argv.ng2 || argv.ng4) {
```

Comment:
nit: we technically don't need this anymore.  I don't think most of the E2E tests will work with 2.x or 4.x either.